workshop/content/basics/templates/flipping-formats-and-cleanup/_index.md

```diff
@@ -0,0 +1,159 @@
+
+{{% notice note %}}
+You can also join strings with the `Fn::Sub` [intrinsic function](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html). Later on in this lab, you will learn how to use `cfn-flip` to convert from `Fn::Join` function declarations into `Fn::Sub`.
+{{% /notice %}}
+
+Example template excerpts, that focus on points you just learned, are summarized next:
+
+```json
+[...]
+    "Parameters": {
+        "ParameterName": {
+            "Description": "Name you wish to specify for your SSM Parameter.",
+[...]
+        },
+        "ParameterValue": {
+            "Description": "Value you wish to specify for your SSM Parameter.",
+[...]
+        }
+    },
+    "Resources": {
+        "MyParameter": {
+            "Type": "AWS::SSM::Parameter",
+            "Properties": {
+                "Name": {
+                    "Ref": "ParameterName"
+                },
+                "Type": "String",
+                "Value": {
+                    "Ref": "ParameterValue"
+                },
+                "Description": {
+                    "Fn::Join": [
+                        " ",
+                        [
+                            "My",
+                            {
+                                "Ref": "ParameterName"
+                            },
+                            "example parameter"
+                        ]
+[...]
+```
+
+Next, use `cfn-flip` to convert your template from the JSON format into YAML. Choose to run the following command from the directory where the `example_parameter.json` JSON-formatted template is located:
+
+```shell
+$ cfn-flip example_parameter.json example_parameter.yaml
+```
+
+{{% notice note %}}
+The `cfn-flip` tool automatically detects the format of your input template (in your case, JSON), and will convert it into the opposite output format accordingly (in this example, into YAML). To explicitly tell `cfn-flip` to convert into YAML or into JSON (and to have `cfn-flip` assume the input template uses the opposite format), choose the `-y` (or `--yaml`) option or the `-j` (or `--json`) option, respectively. For more information, choose to run ` cfn-flip --help` from the command line.
+{{% /notice %}}
+
+As a result, you should now have a new template, in the same directory, called `example_parameter.yaml`. Open the template in your favorite text editor: can you recognize elements that you converted into YAML with `cfn-flip`?
+
+Excerpts from the `example_parameter.yaml` template are shown next. You will note that the `Fn::Join` intrinsic function, that you saw in the JSON-formatted template, is now represented with its YAML short form, `!Join`; this is because `cfn-flip` [makes use of short form function declarations](https://github.com/awslabs/aws-cfn-template-flip#about) where possible:
```

Comment:
```suggestion
Excerpts from the `example_parameter.yaml` template are shown next. You will note that the `Fn::Join` intrinsic function, which you saw in the JSON-formatted template, is now represented in its YAML short form, `!Join`; this is because `cfn-flip` [makes use of short form function declarations](https://github.com/awslabs/aws-cfn-template-flip#about) where possible:
```